docs/source/build_network.rst

```diff
@@ -1,1466 +1,13 @@
 Building Your First Network
 ===========================
 
-.. note:: The Build your first network (BYFN) tutorial has been deprecated. If
+.. note:: The Build your first network (BYFN) tutorial has been removed. If
           you are getting started with Hyperledger Fabric and would like to deploy
```

Comment:
Should we link to versions of BYFN that are in a previous version of Fabric? ie, You can find a version of BYFN in Fabric 2.1? We did that when we first introduced chaincode lifecycle

Reply:
Good idea Nik, link added.